evap/evaluation/templates/email_base.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html style="background-color: #495057;">
+
+<head>
+  <meta charset="utf-8"/>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+  <meta name="viewport" content="width=device-width; initial-scale=1; shrink-to-fit=no"/>
+  <title>EvaP</title>
+  <style>
+    @media (min-width: 768px) {
+      .spacer {
+        width: 15% !important;
+      }
+
+      .email-content {
+        width: 70% !important;
+      }
+    }
+  </style>
+</head>
+
+<body style="margin: 0px; padding: 0px; font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, 'Helvetica Neue', Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'; font-size: 1rem; font-weight: 400; line-height: 1.5; color: #212529; background-color: #ffffff;">
+  <table style="background-color: #495057; width: 100%; border-collapse: collapse;" cellspacing="0" cellpadding="0">
+    <tr>
+      <td>
+        <a href="{{ page_url }}">
+          <img src="https://evap.hpi.de/static/images/evap.png" height="50" style="display: block;"/>
+        </a>
+      </td>
+      <td>
+      </td>
+    </tr>
+  </table>
+
+  <table style="width: 100%; border-collapse: collapse;">
+    <tr style="background-color: #FDFDFD;">
+      <td class="spacer" style="width: 0px;">
+      </td>
+      <td class="email-content" style="padding: 4%; width: 100%">
+        <table style="background-color: white; width: 100%; border: 2px solid #F0F0F0; border-collapse: collapse;">
+          <tr>
+            <td style="text-align: center; font-size: 18px; padding: 1em; background-color: #F8F8F8; border-bottom: 2px solid #F0F0F0">
+              {{ email_subject|linebreaksbr }}
+            </td>
+          </tr>
+          <tr>
+            <td style="padding: 2%">
```

Comment:
regarding the non-fixed width mentioned above, percentage values for padding here and in other places feel a bit strange. how does it look when stretching the window?